subprojects/client-protocol/src/main/java/org/gradle/profiler/client/protocol/Client.java

```diff
@@ -57,15 +55,15 @@ public void send(StudioSyncRequestCompleted message) {
         }
     }
```

Comment:
Why do we have these separate overloads? Couldn't we just have a `send(Message)` method here? Or have a common super-interface for `Message`s that can be passed in here?

Reply:
I believe so you can't send wrong messages the other side. But I wil remove that.